### PR TITLE
Decrease the memory usage by MERRA2 for 6 times

### LINKED ISSUE #902: Decrease the usage of memory 6 times by reading MERRA2 data for 2 months instead of 12 months
## Description
Provide a clear and concise description of what the problem is. 
The MERRA2 aerosol data are read for 12 months and save in the memory, which use tremendous memory and some tie

## Solution
Add a clear and concise description of what you want to happen.
Because the interpolation in time is linear, two months of aerosol data is needed and a way is found to solve the problem

## Alternatives
If applicable, add a description of any alternative solutions or features you've considered.

## Related to
Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related. Example:
- required to support noaa-emc/nems/issues/<issue_number>
- needed by noaa-emc/fv3atm/pull/<pr_number>

### PR DESCRIPTION
**Pull request by AnningCheng-NOAA**
# PR Checklist

- [X ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ X] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ X] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below. 

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Provide a detailed description of what this PR does. What bug does it fix, or what feature does it add? Is a change of answers expected from this PR? Are any library updates included in this PR (modulefiles etc.)?
 read MERRA2 data for two months instead of 12 months and read in the data in interpolation when needed.  The memory usage by MERRA2 decreases 6 times.  There is no results change. no library update. 

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #902 

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel regression tests has been successful in hera
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

-  waiting on https://github.com/NCAR/ccpp-physics/pull/774
-  waiting on https://github.com/NOAA-EMC/fv3atm/pull/423

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
Please fill out the required information in the PR template, including the relevant PRs that you've made in FV3 or CCPP relating to this change.

**Comment by AnningCheng-NOAA** [comment]
Denise, I have just updated the template. Please let me know that I missed
anything.

**Comment by DeniseWorthen** [comment]
Anning, it looks like this also makes a change in ccpp/physics but I don't see it listed here. 

It's also really helpful to have the actual link to the issue or PR in the template. You can do that by using the 'insert link' button and then copying the url from the other repo. Or, within the same repo (for example, in FV3), you can just start typing the words "fixes #" and a list of possible issues and PRs will pop-up for you to chose from. 

Also, can you edit the title of your FV3 PR? It needs to be descriptive of your change and you have "changes made in gitmodules"

**Comment by AnningCheng-NOAA** [comment]
Denise, I do not know how to edit the title of my FV3 PR. Could you give
some instructions?

**Comment by DeniseWorthen** [comment]
On the far right side next to the title is a button that says "edit". You can change the title there.

**Comment by AnningCheng-NOAA** [comment]
Denise, Thank you! I have just changed the title of my PRs.

**Comment by junwang-noaa** [comment]
Please fix the revision of your fv3 branch.

**Review by climbfuji — COMMENTED** [review]
Looks good to me. Will approve after regression testing is complete and the submodule pointer was updated.

**Comment by junwang-noaa** [comment]
@climbfuji All the RT passed. Would you please commit CCPP PR#774? Thanks

**Comment by climbfuji** [comment]
> @climbfuji All the RT passed. Would you please commit CCPP PR#774? Thanks

Done - new ccpp-physics hash is 27a312c

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/AnningCheng-NOAA/fv3atm
```
Comment (climbfuji) [inline .gitmodules]: This still needs to be reverted.

**Review by climbfuji — CHANGES_REQUESTED** [review]
`.gitmodules` needs to be reverted

**Review by junwang-noaa — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]

**Pull request merged**